app/views/friend_requests/received_request.html.erb

```diff
@@ -5,28 +5,35 @@
 
     <div class="col-8">
       <div class="card">
-        <% if current_user.pending_requests_from_users.count.positive? %>
+        <% if current_user.incoming_pending_friends.count.positive? %>
           <div class="card-header d-flex flex-column">
             <span>
-              <strong>Respond to Your <%= pluralize(current_user.pending_requests_from_users.count, 'Friend Request')%> </strong>
+              <strong>Respond to Your <%= pluralize(current_user.incoming_pending_friends.count, 'Friend Request')%> </strong>
               </span>
             <span><%= link_to 'View Sent Request', sent_request_path %></span>
           </div>
           <ul class="list-group list-group-flush">
-            <% current_user.pending_requests_from_users.each do |user| %>
+            <% current_user.incoming_pending_friends.each do |user| %>
               <li class="list-group-item">
                 <div class="post-creator">
                 <span><%= link_to gravatar_for(user, size: 55 ), user_path(user), class: "non-friend-image" %></span>
                   <div class="post-creator-info d-flex justify-content-between align-items-center">
                     <p class="card-subtitle ">
                       <%= link_to user.name, user_path(user) %>
                     </p>
-                    
-                    <%= form_for(current_user.passive_friendships.find_by(passive_friend_id: user.id)) do |f| %>
-                      <%= hidden_field_tag :passive_friend_id, user.id %> 
-                      <%= f.button "Confirm", name: 'accept_request', class: "btn btn-sm btn-blue friend-btn" %>
-                      <%= f.button "Decline Request", name: 'decline_request', class: "btn btn-sm btn-secondary friend-btn", data: {confirm: 'Are you sure?'} %>
-                    <% end %>
+                    <div class="friendship-forms d-flex">
+                      <div class="create-friendship-form mr-1">
+                        <%= form_for(@friendship) do |f| %>
+                          <%= hidden_field_tag :passive_friend_id, user.id %> 
+                          <%= f.submit "Confirm", class: "btn btn-sm btn-blue friend-btn" %>
+                        <% end %>
+                      </div>
+                      <div class="delete-friendship-form ml-1">
+                        <%= form_for(current_user.incoming_pending_requests.find_by(user_id: user.id), html: {method: :delete}) do |f| %>
```

Comment:
**[REQUIRED]**
* Let's separate code logic from the views([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright).